src/extras/PMREMGenerator.js

```diff
@@ -306,7 +308,7 @@ class PMREMGenerator {
 
 	}
 
-	_sceneToCubeUV( scene, near, far, cubeUVRenderTarget ) {
+	_sceneToCubeUV( scene, near, far, cubeUVRenderTarget, position ) {
```

Comment:
I've tested this changes locally and it changes the behavior of `PMREMGenerator` which seems wrong. You can clearly see this with `webgl_loader_gltf_dispersion`.

Making the size and origin of `fromScene()` configurable should not affect current usage.

Reply:
Yes, I have to take a closer look at that. The PMREMGenerator in the src differs in some parts from the one in the node system.
I've been busy analyzing another issue over the last few days #28519. This was then quickly solved by aardgoose with #28523. I then tested that straight away and it works just fine.